ui/src/components/retreat/about/AboutContainer.js

```diff
@@ -0,0 +1,85 @@
+import {
+  Flex,
+  Text,
+  Box,
+  HStack,
+  VStack,
+  Stack,
+  Container,
+  Button,
+  Image,
+} from '@chakra-ui/react';
+import retreatTheme from '../retreatTheme';
+import '@fontsource/sora';
+import '@fontsource/inter';
+
+const AboutContainer = () => {
+  return (
+    <>
+      <Flex
+        w="full"
+        h="100vh"
+        justify="center"
+        bgImage="linear-gradient(90deg, #0FB4BE 33%, rgba(0, 0, 0, 0) 33%), linear-gradient(90deg, #FFDC82 66%, #F39371 66%)"
```

Comment:
Not sure if they changed it after you did this, but looks like the colors aren't matching? Should be:
#A9E0E3, #FFE9AF, #F2BBA8

Reply:
oh i might have copied the old design's colours